frontend/test/pytest/test_aws_braket_devices.py

```diff
@@ -0,0 +1,316 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+
+pytest.importorskip("braket")
+
+import numpy as np
+import pennylane as qml
+
+from catalyst import qjit
+
+
+def test_no_parameters_braket():
+    """Test no-param operations on braket.aws.qubit."""
+
+    def circuit():
+        qml.PauliX(wires=1)
+        qml.PauliY(wires=2)
+        qml.PauliZ(wires=0)
+
+        qml.Hadamard(wires=0)
+        qml.Hadamard(wires=1)
+        qml.Hadamard(wires=2)
+
+        qml.S(wires=0)
+        qml.T(wires=0)
+
+        qml.CNOT(wires=[0, 1])
+        qml.CNOT(wires=[1, 0])
+
+        qml.CZ(wires=[0, 1])
+        qml.CZ(wires=[0, 2])
+
+        qml.SWAP(wires=[0, 1])
+        qml.SWAP(wires=[0, 2])
+        qml.SWAP(wires=[1, 2])
+
+        qml.CSWAP(wires=[0, 1, 2])
+
+        return qml.expval(qml.PauliZ(0) @ qml.PauliZ(1))
+
+    qjit_fn = qjit()(
+        qml.qnode(
+            qml.device(
+                "braket.aws.qubit",
+                device_arn="arn:aws:braket:::device/quantum-simulator/amazon/sv1",
+                wires=3,
```

Comment:
@maliasadi is it possible to also use the Braket _local_ simulator? That is, the one that is part of the Python Braket SDK? The reason I suggest this, is that it will be a lot cheaper (since using SV1 requires credits!) and faster (since no remote API call is made).

It should also be testing the exact same pipeline.

Reply:
https://amazon-braket-pennylane-plugin-python.readthedocs.io/en/latest/devices/braket_local.html

Reply:
:100:

Reply:
Done!